src/Eventso.Subscription/Observing/DeadLetter/PoisonEventHandler.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Eventso.Subscription.Observing.DeadLetter
+{
+    public sealed class PoisonEventHandler<TEvent> : IEventHandler<TEvent>
+        where TEvent : IEvent
+    {
+        private const string PredecessorParkedReason = "Predecessor of event is poison and parked.";
```

Comment:
single place with term `parking`. Its confusing. PredecessorReason = "Predecessor got into DLQ" but i think it can be short as PredecessorReason = "Poisoned stream" if we use stream term for messages with the same key

Reply:
done